TAF/testScenarios/functionalTest/V2-API/core-metadata/provisionwatcher/PATCH.robot

```diff
@@ -12,58 +13,116 @@ ${api_version}    v2
     Then Should Return Status Code "207"
     And Should Return Content-Type "application/json"
     And Item Index 0 Should Contain Status Code "409"
     And Item Index 1 Should Contain Status Code "200"
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    [Teardown]  Delete Multiple Provision Watchers Sample, Profiles Sample And Services Sample
 
 ErrProWatcherPATCH002 - Update provision watcher with empty name
-    Given Create Provision Watchers
-    When Update Provision Watchers
-    Then Should Return Status Code "404"
+    Given Create Provision Watchers And Generate Multiple Provision Watchers Sample For Updating Data
+    And Set To Dictionary  ${provisionwatcher}[1][provisionwatcher]  name=${EMPTY}
+    When Update Provision Watchers ${provisionwatcher}
+    Then Should Return Status Code "400"
     And Should Return Content-Type "application/json"
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    [Teardown]  Delete Multiple Provision Watchers Sample, Profiles Sample And Services Sample
 
 ErrProWatcherPATCH003 - Update provision watcher with empty identifiers
-    Given Create Provision Watchers
-    When Update Provision Watchers
-    Then Should Return Status Code "404"
+    Given Create Provision Watchers And Generate Multiple Provision Watchers Sample For Updating Data
+    And Set To Dictionary  ${provisionwatcher}[1][provisionwatcher]  identifiers=&{EMPTY}
+    When Update Provision Watchers ${provisionwatcher}
+    Then Should Return Status Code "400"
     And Should Return Content-Type "application/json"
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    [Teardown]  Delete Multiple Provision Watchers Sample, Profiles Sample And Services Sample
 
 ErrProWatcherPATCH004 - Update provision watcher with autoEvents but no frequency
-    Given Create Provision Watchers
-    When Update Provision Watchers
-    Then Should Return Status Code "404"
+    ${autoEvents}=  Set autoEvents values  24h  false  DeviceValue_Boolean_RW
+    Given Create Provision Watchers And Generate Multiple Provision Watchers Sample For Updating Data
+    And Set To Dictionary  ${provisionwatcher}[2][provisionwatcher]  autoEvents=${autoEvents}
+    And Set To Dictionary  ${provisionwatcher}[2][provisionwatcher][autoEvents]  frequency=${EMPTY}
+    When Update Provision Watchers ${provisionwatcher}
+    Then Should Return Status Code "400"
     And Should Return Content-Type "application/json"
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    [Teardown]  Delete Multiple Provision Watchers Sample, Profiles Sample And Services Sample
 
 ErrProWatcherPATCH005 - Update provision watcher with autoEvents but no resource
-    Given Create Provision Watchers
-    When Update Provision Watchers
-    Then Should Return Status Code "404"
+    ${autoEvents}=  Set autoEvents values  24h  false  DeviceValue_Boolean_RW
```

Comment:
```suggestion
    ${autoEvents}=  Set autoEvents values  24h  false  ${EMPTY}
```
And remove https://github.com/edgexfoundry/edgex-taf/blob/42b0bbb31671a490b1ac7acc448cd950b10485ff/TAF/testScenarios/functionalTest/V2-API/core-metadata/provisionwatcher/PATCH.robot#L70

Reply:
Updated.